### PR TITLE
Azure Security Center - Adaptive Application Control - Rename 'whitelist'

### PR DESCRIPTION
**Pull request by erelh**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by erelh** [comment]
Hi,
We renamed some of the descriptions & models that had the word "whitelist".
That is why there's a breaking change - some of the models were renamed.

**Comment by erelh** [comment]
@markcowl ?

**Comment by erelh** [comment]
@ChenTanyi @xccc-msft @raych1 @tjprescott @MyronFanQiu can somebody please review ? note that tests fail (only) on "breaking change" because some of the internal models were renamed (no API change)

File: specification/security/resource-manager/Microsoft.Security/stable/2020-01-01/applicationWhitelistings.json [hunk 1]
```diff
@@ -359,9 +359,9 @@
         "VersionAndAboveSignature"
       ]
     },
-    "AppWhitelistingIssue": {
+    "AdaptiveApplicationControlIssue": {
```
Comment (markcowl) [inline specification/security/resource-manager/Microsoft.Security/stable/2020-01-01/applicationWhitelistings.json:362]: These name changes in your schemas will create breaking changes in all generated SDKs.  Please revert.

**Review by markcowl — CHANGES_REQUESTED** [review]

**Comment by markcowl** [comment]
@azuresdkci test this please

**Comment by markcowl** [comment]
@azuresdkci add to whtelist

**Comment by markcowl** [comment]
@azuresdkci *** /openapibot sdkautomation ***

**Review by markcowl — APPROVED** [review]
After discussion, the breaking changes are desirable to remove an offensive term.

**Pull request merged**